_pages/en/strategy-build-vs-buy.md

```diff
@@ -203,18 +293,18 @@ The requirement is that the system level architectural decisions and expertise r
 In contrast to traditional build decisions[^2], when the Government of Canada purchases resources from industry the Government of Canada will be the owner of the product itself -- responsible for the projects implementation and operations; overseeing teams supplied by vendors who provide the technical expertise[^3].This ensures that knowledge about the details and functioning of the system exist within the department[^4].
 Expertise and knowledge regarding implementation details of systems Canadians rely on must exist within the Government of Canada[^5].
 
-#### Alignment Actions
+#### Alignment Actions<!-- omit in toc -->
 
-##### 1. Build Mandate Related Services
+##### 1. Build Mandate Related Services<!-- omit in toc -->
 
-Leadership, policies, technology, and user needs change.
-The department needs to be in full control of its ability to deliver on its mandate[^6].
-As discussed above risk to Canadians cannot be off loaded on to vendors through responsibility offloading (for further discussion please refer to ([Annex 1, Historical Context](#historical-context)).
+Leadership, policies, technology, and user needs change over time.
+The department needs to be in full control of its ability to deliver on its mandate[^6] while adjusting to said changes.
+As discussed above, risk to Canadians cannot be off loaded on to vendors through responsibility offloading (for further discussion please refer to ([Appendix C - Context, Historical Context](#historical-context)).
 To the contrary, attempting to offload responsibility to vendors introduces risk to the government, as such large projects take years to deliver any value to Canadians, and have an increased likelihood of failure[^7].
 
 For further discussion, reference [Annex 1, GC IT Project Analysis](#gc-it-project-analysis).
 
-##### 2. Buy software development services over products
+##### 2. Buy software development services over products<!-- omit in toc -->
```

Comment:
remove from Appendix A (diagnostic). This appendix should only list problem statements (the diagnosis) that is leading us to make policy decisions (the strategy).

This should be a guiding policy rule. It reflects the Cloud First policy and the GC Architecture Standards though. Perhaps it should be positioned in a way that breaks down IT Solutions - IT Products - Software - Infrastructure. Where those development services can be at any layers (which... reflects the SaaS, PaaS, IaaS model really but explains it to non-cloud audience)